charts/opentelemetry-demo/values.yaml

```diff
@@ -381,6 +385,10 @@ components:
         value: "4318"
       - name: OTEL_COLLECTOR_HOST
         value: $(OTEL_COLLECTOR_NAME)
+      - name: FLAGD_HOST
```

Comment:
I don't believe the frontend-proxy (envoy) needs the FLAGD environment variables.

Reply:
It cannot expose flagd to the browser clients (frontend client side flag provider) without those.